packages/models-library/src/models_library/api_schemas_webserver/users_preferences.py

```diff
@@ -0,0 +1,28 @@
+from typing import Any, TypeAlias
+
+from pydantic import BaseModel, Field
+
+from ..user_preferences import PreferenceIdentifier
+
+
+class FrontendUserPreference(BaseModel):
+    default_value: Any = Field(
+        default=..., alias="defaultValue", description="used by the frontend"
+    )
+    value: Any = Field(default=..., description="preference value")
+
+    class Config:
+        allow_population_by_field_name = True
+
+
+FrontendUserPreferencesGet: TypeAlias = dict[
+    PreferenceIdentifier, FrontendUserPreference
+]
+
+
+class FrontendUserPreferencePatchRequestBody(BaseModel):
+    value: Any
+
+
+class FrontendUserPreferencePatchPathParams(BaseModel):
+    preference: PreferenceIdentifier
```

Comment:
this is wrong, it should be `frontend_preference_name` so it matches with the path parameters

In such a simple case, just drop this class and use directly ... 
```python

@router.patch(
    "/me/preferences/{preference_name}",
    status_code=status.HTTP_204_NO_CONTENT,
)
async def set_frontend_preference(preference_name: PreferenceIdentifier, body: FrontendUserPreferencePatchRequestBody ):
    ...
```